docs/DeveloperGuide.adoc

```diff
@@ -570,22 +565,284 @@ image::autocomplete.png[width="500"]
 
 // end::autocompleteDG[]
 
-// tag::addappointment[]
-=== Adding & Storing Appointments
+// tag::find[]
+=== Finding a Contact or Pet Patient
+The find function allows users to find for a specific contact or pet patient through the listed fields.
+
+==== Current Implementation
+
+==== Design Consideration
+
+===== Aspect:
+* **Alternative 1:**
+** Pros:
+** Cons:
+
+* **Alternative 2 (current choice):**
+** Pros:
+** Cons:
+
+==== Current Limitations
+Users are unable to find appointment by type as the CalendarFX runs differently from the other classes.
+
+==== Future Considerations
+//end::find[]
+
+// tag::listappt[]
+=== Listing Appointments of Different Views and Dates
+The command allows users to list appointments based on the date that they wish to see, and the results are displayed via the API of CalendarFX.
+
+==== Current Implementation
+After Medeina implemented CalendarFX, users are able to view appointments at a specified date by clicking on the Calendar. However, as Medeina is a CLI-based application, users should not be required to move their mouse. Thus, there was a need to be able to view appointments of specific dates through the command line.
```

Comment:
It might be better to say "user should be able to jump to certain date to check appointments through command lines only."